src/components/toast/global_toast_list.tsx

```diff
@@ -283,6 +305,45 @@ export const EuiGlobalToastList: FunctionComponent<EuiGlobalToastListProps> = ({
     );
   });
 
+  if (
+    showClearAllButtonAt &&
+    toasts.length >= CLEAR_ALL_TOASTS_THRESHOLD_DEFAULT
```

Comment:
Looks like this is still using the old constant, we need to point at the configurable prop

```suggestion
    showClearAllButtonAt &&
    toasts.length >= showClearAllButtonAt
```

Reply:
Thanks for spotting this.